atd-toolbox/read_replica_proxy/haproxy.cfg.template

```diff
@@ -0,0 +1,16 @@
+global
+  log stdout format raw local0
+
+defaults
+  log global
+  mode tcp
+  timeout connect 5000ms
+  timeout client 50000ms
+  timeout server 50000ms
+
+frontend postgresql_front
+  bind 0.0.0.0:5432
+  default_backend postgresql_back
```

Comment:
This defines what we listen for, which is incoming connections on port `5432` on all addresses. All addresses are used because this is happening inside a docker container which only has `5432` forwarded to it from all NICs on the host.